File: lesson_12/Homework_12_6/controller/VendController.java
package lesson_12.Homework_12_6.controller;

import lesson_12.Homework_12_6.model.DrinkMachine;
import lesson_12.Homework_12_6.model.GetDrink;
import lesson_12.Homework_12_6.product.Drink;

public class VendController {
    private GetDrink get = new GetDrink();
    private DrinkMachine drinks = new DrinkMachine();

    public void getDrink() {
        get.getDrink();
    }

    
    public void setDrink(Drink drink) {
        drinks.setDrink(drink);
    }

public void getMenu(){
    drinks.getMenu();
}

}

File: lesson_13/hometask/HW_py/VendController.py
from Drink import Drink
from DrinkMachine import DrinkMachine
from GetDrink import GetDrink

class VendController(DrinkMachine):

    def getDrink(self):
        GetDrink.getDrink(self)

    def setDrink(self, drink):
        DrinkMachine.setDrink(self, drink)

    def getMenu(self):
        DrinkMachine.getMenu(self)
